agent/config/config.go

```diff
@@ -237,6 +241,25 @@ func get(args []string, cfg *Config, l *logrus.Entry) (configFileF string, err e
 			cfg.Paths.ExportersBase = abs
 		}
 
+		if cfg.Paths.TempDir == "" {
+			wd, err := os.Getwd()
```

Comment:
it should be relative to `cfg.Paths.PathsBase`

Reply:
I will discuss with some heavy users of PMM to see if this is a good choice.

Reply:
other fields works that way, so it should work the same

Reply:
Done, pending validation from MS.